src/main/scala/net/cakesolutions/CakeDockerComposePlugin.scala

```diff
@@ -68,7 +68,9 @@ object CakeDockerComposePlugin extends AutoPlugin {
           projectName ++
           projectOverrides ++
           Seq("up", dockerComposeUpLaunchStyle.value) ++
-          dockerComposeUpExtras.value
+          dockerComposeUpExtras.value,
+        file("."),
+        dockerComposeEnvVars.value.toSeq: _*
```

Comment:
For anyone reading and wondering, this is
`def apply(command: Seq[String], cwd: File, extraEnv: (String, String)*): ProcessBuilder =
    apply(command, Some(cwd), extraEnv: _*)`
on `sbt.Process`.